modules/nw-coredns-egress-firewall.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/configuring-egress-firewall-ovn.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-coredns-egress-firewall_{context}"]
+= Improved DNS resolution and resolving wildcard domain names
+
+There might be situations where the IP addresses associated with a DNS record change frequently or you might want to specify wildcard domain names in your egress firewall policy rules.
+
+:FeatureName: Improved DNS resolution for egress firewall rules
```

Comment:
I would consider moving this snippet after the sentence that starts with "In this situation...." to not break up the explanation of what you're covering in the module. Maybe add it after that sentence, then start a new paragraph with "The `DNSNameResolver` custom resource (CR) stores the following information:"

Reply:
good suggestion!